From 75d87b6bbe69d527a3bcd017f8bef08d9a7b127a Mon Sep 17 00:00:00 2001
From: ayufan <ayufan@ayufan.eu>
Date: Mon, 1 Apr 2013 18:44:51 +0200
Subject: [PATCH] revert perl to 5.10.1

---
 data/PVE/Makefile.am |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/data/PVE/Makefile.am b/data/PVE/Makefile.am
index 7635a78..b886ee7 100644
--- a/data/PVE/Makefile.am
+++ b/data/PVE/Makefile.am
@@ -4,7 +4,7 @@ IPCC.c: IPCC.xs
 	xsubpp IPCC.xs > IPCC.xsc
 	mv IPCC.xsc IPCC.c
 
-CFLAGS = -shared -fPIC -Wall -I/usr/lib/perl/5.14.2/CORE -DXS_VERSION=VERSION
+CFLAGS = -shared -fPIC -Wall -I/usr/lib/perl/5.10.1/CORE -DXS_VERSION=VERSION
 LDADD = $(QB_LIBS)
 
 perlbin_PROGRAMS = IPCC.so
-- 
1.7.2.5

