From d9becc79371f01607dc14908b0fa362d32ee816a Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Frederik=20van=20H=C3=B6vell?=
 <frederik@roodburcht.fvhovell.nl>
Date: Mon, 16 Sep 2013 13:07:13 +0200
Subject: [PATCH] Changes for compilation on x86 / i386.
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Signed-off-by: Frederik van Hövell <frederik@roodburcht.fvhovell.nl>
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 21839ae..bb0c285 100644
--- a/Makefile
+++ b/Makefile
@@ -8,7 +8,7 @@ KVMPACKAGE=pve-qemu-kvm
 KVMDIR=qemu-kvm
 KVMSRC=${KVMDIR}-src.tar.gz
 
-ARCH=amd64
+ARCH=i386
 GITVERSION:=$(shell cat .git/refs/heads/master)
 
 KVM_DEB=${KVMPACKAGE}_${KVMVER}-${KVMPKGREL}_${ARCH}.deb
-- 
1.8.4.rc3

